app/src/main/java/org/p2p/wallet/swap/jupiter/repository/JupiterSwapTokenRepository.kt

```diff
@@ -1,31 +1,53 @@
 package org.p2p.wallet.swap.jupiter.repository
 
+import org.p2p.core.utils.Constants
+import org.p2p.core.utils.orZero
 import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
 import org.p2p.wallet.swap.jupiter.api.SwapJupiterTokensApi
 import org.p2p.wallet.swap.jupiter.api.response.tokens.JupiterTokenResponse
 import org.p2p.wallet.swap.jupiter.repository.model.JupiterToken
+import org.p2p.wallet.user.repository.prices.TokenId
+import org.p2p.wallet.user.repository.prices.TokenPricesRemoteRepository
+import timber.log.Timber
+import java.math.BigDecimal
 import kotlinx.coroutines.withContext
 
 class JupiterSwapTokenRepository(
     private val api: SwapJupiterTokensApi,
+    private val tokenPricesRepository: TokenPricesRemoteRepository,
```

Comment:
Could you rename it plaese? 
The keyword `Remote` is used for implementation class.
`TokenPricesRepository` would be ok